src/AudioToolbox/AudioSession.cs

```diff
@@ -281,16 +283,16 @@ public static void Initialize ()
 			Initialize (null, null);
 		}
 
-		public static void Initialize (CFRunLoop runLoop, string runMode)
+		public static void Initialize (CFRunLoop? runLoop, string? runMode)
 		{
-			CFString s = runMode == null ? null : new CFString (runMode);
+			CFString? s = runMode is null ? null : new CFString (runMode);
 #if NET
 			int k;
 			unsafe {
 				k = AudioSessionInitialize (runLoop.GetHandle (), s.GetHandle (), &Interruption, IntPtr.Zero);
 			}
 #else
-			int k = AudioSessionInitialize (runLoop == null ? IntPtr.Zero : runLoop.Handle, s == null ? IntPtr.Zero : s.Handle, Interruption, IntPtr.Zero);
+			int k = AudioSessionInitialize (runLoop is null ? IntPtr.Zero : runLoop.Handle, s is null ? IntPtr.Zero : s.Handle, Interruption, IntPtr.Zero);
```

Comment:
You can call GetHandle to simplify the code:

```suggestion
			int k = AudioSessionInitialize (runLoop.GetHandle (), s.GetHandle (), Interruption, IntPtr.Zero);
```

GetHandle is an extension method that's safe to call on null:

https://github.com/xamarin/xamarin-macios/blob/3b617adb0f0ad68d95129f550a7672629d46a2c5/src/ObjCRuntime/INativeObject.cs#L25-L28